docs/user/subprojects.rst

```diff
@@ -46,6 +46,18 @@ Adding a custom domain to a subproject is not allowed,
 since your documentation will always be served from
 the domain of the parent project.
 
+Separate release cycles
+-----------------------
+
+When several projects are documented in the same space, it's common that they follow separate release cycles.
```

Comment:
I don't know what this means? What is a `space`? How does it relate to the release cycle?

Reply:
I removed the term `space` which I guess is something I have been indoctrinated to say casually from academia :D